src/ucp/core/ucp_ep.c

```diff
@@ -1524,15 +1524,23 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
     if (get_zcopy_lane_count == 0) {
         /* if there are no RNDV RMA BW lanes that support GET Zcopy, reset
          * min/max values to show that the scheme is unsupported */
-        config->tag.rndv.min_get_zcopy = SIZE_MAX;
-        config->tag.rndv.max_get_zcopy = 0;
+        config->tag.rndv.min_get_zcopy   = SIZE_MAX;
+        config->tag.rndv.max_get_zcopy   = 0;
+        config->tag.rndv.get_zcopy_split = 0;
+    } else {
+        config->tag.rndv.get_zcopy_split = config->tag.rndv.min_get_zcopy <=
+                                           (config->tag.rndv.max_get_zcopy / 2);
```

Comment:
```
if ((get_zcopy_lane_count == 0) ||
   (config->tag.rndv.min_get_zcopy > (config->tag.rndv.max_get_zcopy / 2))) {
        config->tag.rndv.min_get_zcopy   = SIZE_MAX;
        config->tag.rndv.max_get_zcopy = 0;	       
}
```

Reply:
we should not disable GET/PUT Zcopy, since it can be used for messages with the sizes = `[min_get_zcopy..max_get_zcopy]`

Reply:
ok